qrcode/image/svg.py

```diff
@@ -152,6 +153,46 @@ def _write(self, stream):
         super()._write(stream)
 
 
+
+class SvgForPlotters(SvgPathImage):
+    """
+    Draws the modules as concentric squares ; this is made necessary for tools such as plotters
+    and laser cutters which otherwise only draw the outline of the module.
+
+    The stroke_width parameter determines how many concentric squares are drawn.
+    """
+    QR_PATH_STYLE = {'fill': 'none', 'fill-rule': 'nonzero', 
+                     'stroke': '#000000'}
+
+    def __init__(self, *args, **kwargs):
+        self.stroke_width = kwargs.pop('stroke_width', 0.2)
+        self.QR_PATH_STYLE['stroke-width'] = f'{self.stroke_width}'
+        super().__init__(*args, **kwargs)
+
+
+    def _generate_subpaths(self):
+        """Generates individual QR points as subpaths"""
+
+        rect_size = self.units(self.box_size, text=False)
+
+        for point in self._points:
+            x_base = self.units(
+                (point[0]+self.border)*self.box_size, text=False)
+            y_base = self.units(
+                (point[1]+self.border)*self.box_size, text=False)
+
+            for x in range(1,int(float(rect_size)/self.stroke_width/2)+1):
+                if x%2==1:
```

Comment:
Seems like you should range with `step` instead?

Reply:
I have never used `step` nor have I found a python reference to `step` that seems to make sense.

this for loop works. what would be the advantage of using `step` versus my proposition?

Reply:
It's just prettier code, no need for the conditional.
https://docs.python.org/3/library/functions.html#func-range

Reply:
oh I see. this totally makes sense.